src/main/java/net/pms/formats/MKV.java

```diff
@@ -67,7 +52,6 @@ public boolean ps3compatible() {
 			"rmv",
 			"rmvb",
 			"rm",
-			"ty",
```

Comment:
@Sami32 Because it's also listed in ```MPG.java``` which makes it a bug. I had to decide which one it matched best with, and from what I could understand that was mpeg.